README.md

```diff
@@ -128,7 +128,8 @@ Use `assign` to set the context
 
 ~~~elixir
 setup context do
-  Dict.put context, :necessary_key, :necessary_value
+  Dict.put(context, :necessary_key, :necessary_value)
+    |> Dict.put :many_key, ~W(white space delimited no interp)
```

Comment:
I did consider that example. By all means this is quite cosmetic so feel free to pass.  The other point that was confusing for me as I learn testing in Elixir (and the language) was what is actually happening in `setup context do`.  I was looking for something like a let (from RSpec - not that RSpec is the only/right way) because the syntax started to seem similar.  I totally missed on first pass that context is assigned the return value and I have to put context into my test.  An Erlangist probably would have never been confused (immutability first logic)

What should the practice be if I need to setup something like file IO and a mixture of simple test assertion variables? I can put them all into a map and access them. I get a little annoyed writing  context.test_variable context.another_test_variable all over, but Ruby can make you lazy. Also is there a way to automatically bring into scope the context variable per test that I am missing or do I have to explicitly pass it? Perhaps I haven't rtfm'd?